README.md

```diff
@@ -75,12 +93,74 @@ The `onepassword.Item` model represents Items and `onepassword.Vault` represent
 The `connect.Client` also supports methods for:
 
 - listing Vaults
+```go
+vaults, err := client.GetVaults()
+if err != nil {
+	panic(err)
+}
+```
 - listing items in a Vault
+```go
+items, err := client.GetItems({vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - searching by Item Title
+```go
+items, err := client.GetItemsByTitle({item_title}, {vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - Retrieving Item by Vault and Item UUID
+```go
+item, err := client.GetItem({item_uuid}, {vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - Creating Items in a Vault
+```go
+item := &onepassword.Item{
+	Fields: []*onepassword.ItemField{{
+		Value: "mysecret",
+		Type: "STRING",
+	}},
+	Tags:     []string{"1password-connect"},
+	Category: onepassword.Login,
+	Title:    "Secret String",
+}
+
+postedItem, err := client.CreateItem(item, {vault_uuid})
+if err != nil {
+	panic(err)
+}
+```
 - Updating Items
+```go
+item2 := &onepassword.Item{
```

Comment:
I think this example should be updating an existing item but it just creates a second one.